shell/platform/tizen/tizen_view_nui.h

```diff
@@ -0,0 +1,64 @@
+// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef EMBEDDER_TIZEN_VIEW_NUI_H_
+#define EMBEDDER_TIZEN_VIEW_NUI_H_
+
+#include <dali-toolkit/public-api/controls/image-view/image-view.h>
+#include <dali/devel-api/adaptor-framework/event-thread-callback.h>
+#include <dali/devel-api/adaptor-framework/native-image-source-queue.h>
+#include <dali/devel-api/common/stage.h>
```

Comment:
Does `devel-api` mean that the APIs are not finalized and subject to change in the future?

Reply:
As far as I know the devel-api is for inhouse app developers. it didn't ACR.
The APIs we use are open as DALI_ADAPTOR_API.

> subject to change in the future?

It could change, but I think it's unlikely.